src/rust/lib/prelude/src/macros.rs

```diff
@@ -105,6 +105,15 @@ macro_rules! alias {
     )*};
 }
 
+#[macro_export]
+macro_rules! alias2 {
+    ($(#$meta:tt)* $name:ident = $($tok:tt)*) => {
+        $(#$meta)*
+        pub trait $name: $($tok)* {}
+        impl<T:$($tok)*> $name for T {}
+    };
+}
+
```

Comment:
What is alias2? No docs here.

Reply:
left accidentally, thanks for catching it